### PR TITLE
inet_csum: remove ng_ prefix

### PR DESCRIPTION
**Pull request by authmillenon**
And another one of those

```bash
git mv sys/include/net/ng_inet_csum.h sys/include/net/inet_csum.h
git mv sys/net/crosslayer/ng_inet_csum/ sys/net/crosslayer/inet_csum/
git mv sys/net/crosslayer/inet_csum/ng_inet_csum.c sys/net/crosslayer/inet_csum/inet_csum.c
git grep --name-only -i 'ng_inet_csum' | xargs sed -i -e 's/ng_\(inet_csum\)\(.\+\)\( \+\/.*\)/\1\2   \3/gI' -e 's/\(#define *\)ng_\(inet_csum\)\([^ ]\+\)\( \+\)/\1\2\3   \4/gI' -e 's/ng_\(inet_csum\)/\1/gI'
```

### CODE REVIEW

File: sys/include/net/ng_ipv6/hdr.h [hunk 1]
```diff
@@ -287,15 +287,15 @@ static inline uint32_t ng_ipv6_hdr_get_fl(const ng_ipv6_hdr_t *hdr)
  * @return  The non-normalized Internet Checksum of the given IPv6 pseudo header.
  */
 static inline uint16_t ng_ipv6_hdr_inet_csum(uint16_t sum, ng_ipv6_hdr_t *hdr,
-                                             uint8_t prot_num, uint16_t len)
+        uint8_t prot_num, uint16_t len)
```
Comment (OlegHahm) [inline sys/include/net/ng_ipv6/hdr.h]: unrelated and unwanted(?)

Comment (OlegHahm) [inline sys/include/net/ng_ipv6/hdr.h]: plus: missed to change right thing

Comment (OlegHahm) [inline sys/include/net/ng_ipv6/hdr.h]: now I don't trust sed any more...

Comment (authmillenon) [inline sys/include/net/ng_ipv6/hdr.h]: will fix.

Comment (authmillenon) [inline sys/include/net/ng_ipv6/hdr.h]: what right thing?

Comment (OlegHahm) [inline sys/include/net/ng_ipv6/hdr.h]: ng_ipv6_hdr_inet_csum -> ipv6_hdr_inet_csum

Comment (authmillenon) [inline sys/include/net/ng_ipv6/hdr.h]: this is not topic of this PR. Especially `ng_ipv6_hdr` needs some disentanglement (the header build functions are gnrc, the header and some related function as this one are not).

Comment (authmillenon) [inline sys/include/net/ng_ipv6/hdr.h]: Anyway: fixed the indentation error.

Comment (OlegHahm) [inline sys/include/net/ng_ipv6/hdr.h]: Oops, my bad. Didn't read carefully (the part between ng_ and inet_csum was lost ;)).

**Comment by OlegHahm** [comment]
Trust in sed restored, trust in my eyes lowered. ACK! Squash please.

**Comment by authmillenon** [comment]
Squashed

**Comment by authmillenon** [comment]
Rebased

**Comment by authmillenon** [comment]
Oops, am not used to see strider green. But let's wait for travis' verdict before reverting this :-)

**Comment by authmillenon** [comment]
\o/ everything is still allright :-)

**Pull request merged**